translator/totomlconfig/sampleConfig/complete_windows_config.conf

```diff
@@ -1,6 +1,6 @@
 [agent]
   collection_jitter = "0s"
-  debug = false
+  debug = true
```

Comment:
Do we have any tests where debug is false? Is there a way we can test to validate that the log level gets set to WARN or OFF, etc.?

Reply:
I wanted to have minimal changes to the behavior with this change. If debug isn't present in the agent section of the JSON config, then it will be set to false in the TOML config. This can be seen in most of the other tests.

When debug is set to true, the env-config.json will have "CWAGENT_LOG_LEVEL": "DEBUG" present. If debug is set to false, then the "CWAGENT_LOG_LEVEL" key isn't added. If the environment variable is not present or is empty, the log level defaults to "INFO". This is consistent with the current behavior.